Gemfile

```diff
@@ -33,10 +33,10 @@ gem 'druid-tools'
 gem 'moab-versioning', '~> 2.0'
 
 # Database
-group :integration, :staging, :production do
+group :production do
   gem 'ruby-oci8'
 end
-group :test, :local, :development do
```

Comment:
I also see `:local` in `config/database.rb`, but this may be a relic of past usage. Perhaps @ndushay and @jmartin-sul can say more about this one.

Reply:
again, gemfile groups are not the same as environments. This doesn't matter.